inc/site-editor.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Customizations for the site editor.
+ *
+ * @package Create_WordPress_Theme
+ */
+
+namespace Create_WordPress_Theme\Site_Editor;
+
+use Alley\WP\Caper;
+
+add_action( 'after_setup_theme', __NAMESPACE__ . '\action__after_setup_theme' );
+
+/**
+ * Prevents access to the Site Editor on environments other than local and develop.
+ */
+function action__after_setup_theme(): void {
+	global $pagenow;
+
+	if ( ! in_array( wp_get_environment_type(), [ 'local', 'development' ], true ) ) {
+		// Caper is being installed via Create WordPress Project. If it's not available, bail.
+		if ( ! class_exists( 'Alley\WP\Caper' ) ) {
+			return;
+		}
+
+		Caper::deny_to_all()->primitives( 'edit_theme_options' );
+		// Create a custom capability for menu items that should be displayed.
+		Caper::grant_to( 'administrator' )->primitives( 'cwt_edit_menus' );
```

Comment:
We should spell out `create_wordpress_theme` here so that find and replace will update it to the project-specific thing:
```suggestion
		Caper::grant_to( 'administrator' )->primitives( 'create_wordpress_theme_edit_menus' );
```